Updated Pricing (#585)
cirruslabs/cirrus-ci-docs: docs/pricing.md

1. For private personal repositories there is a [very affordable $10 a month plan](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTI=#pricing-and-setup) with 
   access to community clusters for [Linux](guide/linux.md), [Windows](guide/windows.md) and [macOS](guide/macOS.md) workloads.

> 2. Buy [compute credits](#compute-credits) to access managed and pre-configured community clusters for [Linux](guide/linux.md),  [FreeBSD](guide/FreeBSD.md), [Windows](guide/windows.md) and [macOS](guide/macOS.md) workloads.

Reviewer: ```suggestion
2. Buy [compute credits](#compute-credits) to access managed and pre-configured community clusters for [Linux](guide/linux.md), [FreeBSD](guide/FreeBSD.md), [Windows](guide/windows.md) and [macOS](guide/macOS.md) workloads.
```

3. Configure access to [your own infrastructure](#compute-services) and [pay $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup)
   fee for orchestrating your CI workloads on your infrastructure.

> Here is a pricing model of Cirrus CI:

Reviewer: ```suggestion
Here the a pricing model of Cirrus CI:
```

Author: Did you mean 

```suggestion
Here is the pricing model of Cirrus CI:
```

Reviewer: Yeah

**Pros** of this approach:
  
* Use the same pre-configured infrastructure as the Open Source community is enjoying.

> * No need to configure anything. Let Cirrus CI team to manage and upgrade infrastructure for you.

Reviewer: ```suggestion
* No need to configure anything. Let Cirrus CI's team manage and upgrade infrastructure for you.
```

    task:
      use_compute_credits: $CIRRUS_BRANCH == 'master'
    ```

## Compute Services

> Configure and connect one or several [compute services](guide/supported-computing-services.md) to Cirrus CI and [pay $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup) 

Reviewer: ```suggestion
Configure and connect one or more [compute services](guide/supported-computing-services.md) to Cirrus CI and [pay $10/seat/month](https://github.com/marketplace/cirrus-ci/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTM=#pricing-and-setup) 
```

for orchestrating CI workloads on these compute services. 

**Pros** of this approach:

> * Full control of underlying infrastructure. Any type of VMs and containers with any amount of CPUs and memory.

Reviewer: ```suggestion
* Full control of underlying infrastructure. Use any type of VMs and containers with any amount of CPUs and memory.
```

> * Most secure. Setup any firewall and access rules.

Reviewer: I'm no lawyer, but I don't think assurances like this are the best legal-wise.
```suggestion
* More secure. Setup any firewall and access rules.
```

* Pay for CI within your existing cloud and GitHub bills. 
  
**Cons** of this approach:

* Need to configure and connect one or several [compute services](guide/supported-computing-services.md). Might be
  nonintuitive for cases like Anka Build Cloud for macOS.
* Might not be worth the effort for a small team.

> * Need to buy $10/seat/month plan.

Reviewer: ```suggestion
* Need to pay $10/seat/month plan.
```

!!! info "What is a seat?"

> 

Reviewer: ```suggestion
```

>     Seat is simply a GitHub user that initiates CI builds by pushing commints and/or creating pull requests in a **private** repository. 

Reviewer: ```suggestion
    A seat is simply a GitHub user that initiates CI builds by pushing commits and/or creating pull requests in a **private** repository. 
```

>     It can be a real person or a bot. If you are using [Cron Builds](guide/writing-tasks.md#cron-builds) or creating builds thourgh [Cirrus API](api.md)

Reviewer: ```suggestion
    It can be a real person or a bot. If you are using [Cron Builds](guide/writing-tasks.md#cron-builds) or creating builds through [Cirrus's API](api.md)
```

>     it will be counted as an additional seat like a bot.

Reviewer: ```suggestion
    it will be counted as an additional seat (like a bot).
```

>     For example, there are 10 people in your GitHub Organization and only 5 of them are working on several private repositories 

Reviewer: ```suggestion
    For example, if there are 10 people in your GitHub Organization and only 5 of them are working on private repositories 
```

>     where Cirrus CI is configured. The remaining 5 people are working on public repositories or not pushing changes at all. 

Reviewer: ```suggestion
    where Cirrus CI is configured, the remaining 5 people are working on public repositories or not modifying any repositories at all. 
```

>     Let's say [dependabot](https://dependabot.com/) is also configured for these private repositories. 

Reviewer: ```suggestion
    Let's say [Dependabot](https://dependabot.com/) is also configured for these private repositories. 
```
